manifests/feature/idomysql.pp

```diff
@@ -7,6 +7,10 @@
 # [*ensure*]
 #   Set to present enables the feature ido-mysql, absent disables it. Defaults to present.
 #
+# [*ido_mysql_package_ensure*]
+#   Specifies the package version to install. Can be 'installed', 'absent',
```

Comment:
Maybe the name package_ensure is better, because this parameter is part of class "icinga2::feature::**idomysql**". Same for pgsql

We can also save two parameter definitions in params.pp